composer.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name":         "knplabs/gaufrette",
+    "name":         "FaustineLarmagna/gaufrette",
```

Comment:
This must not be part of the branch you use to submit the pull -request. Open the PR using a feature branch containing the wanted changes but not the package renaming